doc/install/index.rst

```diff
@@ -1,17 +1,22 @@
 DAGMC installation overview
 ===========================
 
-This document describes the DAGMC installation process. Before you begin, you
-should know that the following will be required:
+This document describes the DAGMC installation process. 
+
+If you plan to use OpenMC_, you can follow their installation instructions with
+DAGMC built-in.
```

Comment:
Might be good to link to the instructions here: https://docs.openmc.org/en/stable/quickinstall.html